src/hotspot/cpu/aarch64/codeBuffer_aarch64.cpp

```diff
@@ -26,6 +26,61 @@
 #include "asm/codeBuffer.inline.hpp"
 #include "asm/macroAssembler.hpp"
 
+void CodeBuffer::share_trampoline_for(address dest, int caller_offset) {
```

Comment:
Shouldn't it be `shared_trampoline_for` to be consistent with `shared_stub_to_interp_for`?